x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -373,13 +374,11 @@ export const ProcessEventDot = styled(
   )
 )`
   position: absolute;
-  display: block;
   text-align: left;
   font-size: 10px;
   user-select: none;
   box-sizing: border-box;
   border-radius: 10%;
-  padding: 4px;
```

Comment:
:grey_question: A lot of these style rules were set to meet design's guidance via the mocks and signed off by them. You tested to make certain this doesn't alter what they've signed off on?

Reply:
sorry, but we can't support having a padding w/o adjusting the logic that measures the size of the resolver map.